testdata/cloud/pvc-34718.yml

```diff
@@ -7,4 +7,4 @@ spec:
   - ReadWriteOnce
   resources:
     requests:
-      storage: 1Gi
+      storage: 20Gi
```

Comment:
Is this change still a must ? Smaller size should save us some storage cost.

Reply:
yes,  have a look at - `Capacity of a single standard SSD` https://www.alibabacloud.com/help/en/doc-detail/25412.htm#BlockStorageQuota

Reply:
If the minimal size of storage on AliCloud is 20Gi, then we should also get 20Gi even if we use 1Gi (no matter the size as long as it is smaller than 20Gi) there. But it's a wast of money on all the other IAAS